flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mongodb-cdc/src/test/java/org/apache/flink/cdc/connectors/mongodb/table/MongoDBTableFactoryTest.java

```diff
@@ -125,6 +129,9 @@ public class MongoDBTableFactoryTest {
     private static final boolean SCAN_NEWLY_ADDED_TABLE_ENABLED_DEFAULT =
             SCAN_NEWLY_ADDED_TABLE_ENABLED.defaultValue();
 
+    private static final AssignStrategy SCAN_CHUNK_ASSIGN_STRATEGY_DEFAULT =
```

Comment:
Could we add test in MongoDBConnectorITCase for DESCENDING_ORDER AssignStrategy? This can check the result correctness for the new assign strategy.